tools/testbench/common_test.c

```diff
@@ -58,7 +58,7 @@ struct ipc_data {
 	struct ipc_data_host_buffer dh_buffer;
```

Comment:
Merge artifacts in commit message ("#Conflicts").

Reply:
This is still here.

Reply:
@kv2019i I just fixed two commit messages, removed Conflicts text from end. No other changes done.